### DISCUSSION TITLE
Code of Conduct: need community leaders

### DISCUSSION THREAD

**Discussion by getify** [getify/youperiod.app#39] [General]
Please see my note here:

https://github.com/getify/youperiod.app/pull/10#issuecomment-1174441718

If this is interesting to you, please speak up, or reach out to me privately getify@gmail.com.

**Comment by novonimo** [comment]
> Please see my note here:
> 
> [#10 (comment)](https://github.com/getify/youperiod.app/pull/10#issuecomment-1174441718)
> 
> If this is interesting to you, please speak up, or reach out to me privately [getify@gmail.com](mailto:getify@gmail.com).

I proudly announce my readiness for this work

**Reply by getify** [reply]
Can you list your full name and email / contact info here so I can include it?

**Reply by getify** [reply]
ping @novonimo

**Reply by novonimo** [reply]
Nima Mohamadian

nimamohamadian89@gmail.com

**Comment by AbednegoTM** [comment]
✋ I am interested.

**Reply by getify** [reply]
Can you list your full name and email / contact info here so I can include it?

**Reply by AbednegoTM** [reply]
Name: Abednego Mwanza
email: abednegomwanza@gmail.com

**Comment by AbednegoTM** [comment]
If you have been selected as community leader , you can comment on this PR - #10  so that it can be updated and merged.

**Comment by violetadev** [comment]
hey there, I'm interested

**Comment by getify** [comment]
I also heard privately from Prabhpreet Singh volunteering to be part of the CoC leaders group.

**Comment by getify** [comment]
I think it's important that the leaders group have multiple women involved. Can those of you here who are interested please share if you identify as a woman, or in other traditionally under-represented groups?

I'm a white male (as my avatar depicts).

**Comment by aditiranka27** [comment]
I am interested. Count me in!

**Reply by aditiranka27** [reply]
@getify  here is my email for any communication purpose: aditi.ranka@gmail.com

**Comment by violetadev** [comment]
> I think it's important that the leaders group have multiple women involved. Can those of you here who are interested please share if you identify as a woman, or in other traditionally under-represented groups?
> 
> I'm a white male (as my avatar depicts).

I'm female (she/her), latina and I'm an atheist.

**Reply by violetadev** [reply]
artnerdnet@gmail.com

**Reply by getify** [reply]
Can you list your full name here so I can include it?

**Reply by getify** [reply]
ping @artnerdnet

**Comment by prabhpreet-code** [comment]
> I think it's important that the leaders group have multiple women involved. Can those of you here who are interested please share if you identify as a woman, or in other traditionally under-represented groups?
> 
> I'm a white male (as my avatar depicts).

I'm a brown male and my religion is Sikhism.

**Reply by prabhpreet-code** [reply]
My email is prabhpreetsingh.singh0203@gmail.com

**Reply by getify** [reply]
Can you include your full name for proper listing?

**Reply by prabhpreet-code** [reply]
Prabhpreet Singh

**Comment by sandramolina** [comment]
I happy joining in the CoC reinforcement group if there's still room. I am a latina, cisgender woman.
My email is molina.sandra.101@gmail.com

**Comment by pselle** [comment]
👋 I'm interested in helping! I'm a white, queer, cis woman. pam @ thewebivore.com is a good email for listing.

**Comment by getify** [comment]
OK, this is fantastic. Thank you to all of you for speaking up!

In my estimation, I think we now have a diverse enough group of voices that we can feel confident that we can protect this project through careful and judicious guidance and enforcement of the CoC.

Each person here, if we don't yet have it in the thread, we need your publicly listable email address (or other contact info, if you prefer) for Code of Conduct related communications (should someone feel the need to reach out to you privately). Please post or edit here to include that info.

Then later today, let's aim to get these names into the official document and finalize that PR.

----

I am appointing @pselle as another core contributor to the project, in case there are administrative-level things (like moderation) that are needed and I am unavailable to help. So please look to Pam as another source of official project-level guidance or access as you may see the need.

**Comment by getify** [comment]
I've merged/posted our code-of-conduct: https://github.com/getify/youperiod.app/blob/main/CODE_OF_CONDUCT.md

Thank you to those of you who've stepped up to help lead our community!

**Comment by RA9** [comment]
If it is not still  late, I am also interested.

**Reply by getify** [reply]
Appreciate the interest. I think right now, having 8 folks listed in this role is more than enough. But please do help contribute to our community by helping make sure all our discussions are abiding by our values and guidelines, and bring any issues to the attention of these listed folks. Really, it's all our responsibility to self-monitor. :)
